### PR TITLE
(chore) clinical dashboard cleanups

### PR DESCRIPTION
**Pull request by pirupius**
## Requirements

- [x] This PR has a title that briefly describes the work done including the ticket number. If there is a ticket, make sure your PR title includes the ticket number in the format `OHRI-123 My PR title`.
- [ ] My work includes tests or is validated by existing tests.

## Summary
This PR has a collection of refactors to clean up some of the codebase relating to our clinical dashboards 

## Screenshots
NO visible changes

### CODE REVIEW

File: packages/esm-commons-lib/src/api/api.ts [hunk 1]
```diff
@@ -179,6 +179,7 @@ export function fetchPatientComputedConcept_HIV_Status(patientUUID: string) {
 }
 
 export function fetchPatientLastEncounter(patientUuid: string, encounterType) {
+  console.log('fetchPatientLastEncounter', patientUuid, encounterType);
```
Comment (ODORA0) [inline packages/esm-commons-lib/src/api/api.ts:182]: @pirupius You may need to remove this log

**Review by ODORA0 — CHANGES_REQUESTED** [review]

File: packages/esm-commons-lib/src/api/api.ts (hunk 1 above)
Comment (pirupius) [reply packages/esm-commons-lib/src/api/api.ts:182]: Definitely. Don't know how i missed that

Comment (pirupius) [reply packages/esm-commons-lib/src/api/api.ts]: Done

**Review by ODORA0 — APPROVED** [review]
LGTM

**Pull request merged**